docs/configuration/index.md

```diff
@@ -1769,14 +1769,23 @@ If there is an L1 miss and L2 hit, it will also populate L1.
 
 This section describes configurations that control behavior of Druid's query types, applicable to Broker, Historical, and MiddleManager processes.
 
-### Query vectorization config
+### Overriding default query context values
```

Comment:
Please add some example in this section.

Reply:
Done